languages/clojure/exercises/concept/basic-functions/.doc/hints.md

```diff
@@ -0,0 +1,20 @@
+### 1. Create a function that returns 42
+
+You can use the `defn` macro to define a function!
+
+The defn macro accepts the name of the function you are defining, and optional docstring, a vector of input arguments, and the expressions to be evaluated when the function is called!
+
+
+### 2. Create a function that accepts an input and adds 42 to it 
+
+You can name the argument passed into the function by writing its name within the argument vector when the function is defined. The argument can then be referenced by that name anywhere within the function.
+
+Remember to use prefix notation with the '+' function. For example, in Clojure "one plus two" would be written as:
+
+```clojure
```

Comment:
In general, we want the hints to only contain code when there is no other option. I think you could lose the example here, as the "Remember to use prefix notation with the '+' function" should give the student enough context that the student can solve it themselves.

Reply:
removed the code example here. 👍